src/api/consumer/lib/export.ts

```diff
@@ -366,3 +372,19 @@ async function _throwForModified(consumer: Consumer, ids: BitIds) {
     }
   });
 }
+
+function _throwForUnsnappedLaneReadme(lane: Lane) {
+  const readmeComponent = lane.readmeComponent as LaneReadmeComponent;
+
+  const isValid =
+    readmeComponent?.head &&
+    lane.getComponent(readmeComponent.id) &&
+    lane.getComponentHead(readmeComponent.id)?.isEqual(readmeComponent?.head);
+
+  if (!isValid) {
+    throw new Error(
```

Comment:
Normally, when it's a user-error, it's preferable to use `BitError`. This way, we could differentiate them with real errors we want to track in Sentry. (implemented on legacy, but needs to be completed on Harmony).